Gemfile

```diff
@@ -85,7 +85,7 @@ group :test, :development do
   # Pretty printed test output
   gem 'turn', '~> 0.8.3', :require => false
   gem 'fuubar'
-  gem 'rspec-rails'
+  gem 'rspec-rails', '~> 2.99.0'
```

Comment:
Any particular reason we are upgrading these gems? I'm all for upgrading to new versions, but just wondering if there was a particular issue that prompted the upgrade?